internal/rpcserver/rpcserver.go

```diff
@@ -476,6 +480,18 @@ func newWorkState() *workState {
 	}
 }
 
+// isTreasuryAgendaActive returns if the treasury agenda is active or not.
+func isTreasuryAgendaActive(s *Server) (bool, error) {
```

Comment:
Consider making this a member function of `server`, the signature would become: `func (s *Server) isTreasuryAgendaActive() (bool, error)`

Reply:
I did consider it but I tried to remain close to the style surrounding this code. If you feel strongly about this please submit a PR after this PR lands.